### PR TITLE
add runConfiguration for deploying to WebAppOnLinux

### PR DESCRIPTION
**Pull request by Eskibear**

### CODE REVIEW

File: Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/AzureMvpModel.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    public static AzureMvpModel getInstance() {
+        return SingletonHolder.INSTANCE;
+    }
+
+    Map<String, Subscription> subscriptionIdToSubscriptionMap;
+    Map<String, SubscriptionDetail> subscriptionIdToSubscriptionDetailMap;
+    Map<String, List<ResourceGroup>> subscriptionIdToResourceGroupMap;
+
+    private AzureMvpModel(){
+        subscriptionIdToSubscriptionMap = new HashMap<>();
```
Comment (adashen) [inline Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/AzureMvpModel.java]: use ConcurrentHashMap

File: Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/AzureMvpModel.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        subscriptionIdToSubscriptionDetailMap = new HashMap<>();
```
Comment (adashen) [inline Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/AzureMvpModel.java]: same as above

File: Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/AzureMvpModel.java [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        subscriptionIdToResourceGroupMap = new HashMap<>();
```
Comment (adashen) [inline Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/AzureMvpModel.java]: same as above

File: Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/webapp/AzureWebAppMvpModel.java [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    private static final class SingletonHolder {
+        private static final AzureWebAppMvpModel INSTANCE = new AzureWebAppMvpModel();
+    }
+
+    public static AzureWebAppMvpModel getInstance() {
+        return SingletonHolder.INSTANCE;
+    }
+
+    private AzureWebAppMvpModel() {
+        subscriptionIdToWebAppsOnLinuxMap = new HashMap<>();
```
Comment (adashen) [inline Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/webapp/AzureWebAppMvpModel.java]: cocurrenthashmap

File: Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/webapp/AzureWebAppMvpModel.java [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+        subscriptionIdToWebAppsMap = new HashMap<>();
```
Comment (adashen) [inline Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/webapp/AzureWebAppMvpModel.java]: same as above

**Comment by Eskibear** [comment]
retest this please

**Pull request merged**